browser/app/less/inc/misc.less

```diff
@@ -75,6 +75,10 @@
 			border-color: darken(@input-border, 5%);
 		}
 	}
+
+	svg{
+		margin-left: 30%;
+	}
```

Comment:
Thank you very much for this very cool PR @yolossn! One thing I have noticed is that on a screen width lower than 400px and when the width is between 400px and 767px, the QR code is slightly off-centered. We might want to do something like the following to center it on all screen widths:

```suggestion
	svg {
	  display: block;
		margin: 0 auto 5px;
	}
```

Please refer to the images below:

![Screen Shot 2021-03-10 at 8 49 09 PM](https://user-images.githubusercontent.com/27314853/110737565-74606b00-81e2-11eb-928c-2c001e89bb06.png)

![Screen Shot 2021-03-10 at 8 44 44 PM](https://user-images.githubusercontent.com/27314853/110737294-f7cd8c80-81e1-11eb-950e-ae50926751c6.png)